lib/tasks/facts.rake

```diff
@@ -0,0 +1,17 @@
+namespace :facts do
+  desc <<-END_DESC
+Removes facts without any values
+
+This is useful when you used custom facts of different facts sourcers that you don't use anymore.
+After cleaning of such facts they should no longer appear in auto-completion suggestions.
+
+Examples:
+  # foreman-rake facts:clean
+END_DESC
+
+  task :clean => :environment do
+    puts 'Starting orphaned facts clean up...'
+    count = FactCleaner.new.clean!
+    puts "Finished, cleaned #{count} facts"
```

Comment:
After the discussion on IRC, the motivation was rather the count returned by clean! than unifying output.

The convention I know from Programming Ruby book is "Methods that are "dangerous," or modify the receiver, might be named with a trailing "!". but I it's vague and especially in Rails it's used in other contexts. If you didn't expect that, others might be surprised as well so I'll add explicit reader.